tests/src/Common/CoreFX/CoreFX.depproj

```diff
@@ -165,12 +164,6 @@
       <Version>$(MicrosoftDotnetPlatformAbstractionsVersion)</Version>
```

Comment:
Are all the other manual dependencies in this file still needed?

Reply:
Unfortunately there is still some work to do to represent non-platform references as app-local in corefx. Currently I only special case CoreFx.Private.TestUtilities as app-local as it's included in every test assembly. Meanwhile you can reference the `CoreFx.Private.OOB` meta package for all the OOB references.